demisto_sdk/commands/validate/file_validator.py

```diff
@@ -945,17 +956,19 @@ def get_error_ignore_list(self, pack_name):
                     config = ConfigParser(allow_no_value=True)
                     config.read(pack_ignore_path)
 
+                    # create pack ignored errors list
                     if 'demisto-sdk' in config:
+                        ignored_errors_list['pack'] = []
                         for key in config['demisto-sdk']:
```

Comment:
maybe let's change that to pack_ignore?

Reply:
won't it be confusing because of the file name?